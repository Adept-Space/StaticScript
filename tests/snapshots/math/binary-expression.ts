
{
    function equalTwoNumbers(left: number, right: number): void {
        console_log("equalTwoNumbers");
        console_log(left);
        console_log(right);

        if (left == right) {
            console_log("true");
        } else {
            console_log("false");
        }

        console_log("");
    }

    function compareNumberLeftLessRight(left: number, right: number): void {
        console_log("compareNumberLeftLessRight");
        console_log(left);
        console_log(right);

        if (left < right) {
            console_log("true");
        } else {
            console_log("false");
        }

        console_log("");
    }

    function compareNumberLeftMoreRight(left: number, right: number): void {
        console_log("compareNumberLeftMoreRight");
        console_log(left);
        console_log(right);

        if (left > right) {
            console_log("true");
        } else {
            console_log("false");
        }

        console_log("");
    }

    equalTwoNumbers(1, 1);
    equalTwoNumbers(1, 0);

    compareNumberLeftLessRight(1, 100);
    compareNumberLeftLessRight(101, 100);

    compareNumberLeftMoreRight(101, 100);
    compareNumberLeftMoreRight(1, 100);
}
