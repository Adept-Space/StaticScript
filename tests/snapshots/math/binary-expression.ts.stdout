equalTwoNumbers
1.000000
1.000000
true

equalTwoNumbers
1.000000
0.000000
false

compareNumberLeftLessRight
1.000000
100.000000
true

compareNumberLeftLessRight
101.000000
100.000000
false

compareNumberLeftMoreRight
101.000000
100.000000
true

compareNumberLeftMoreRight
1.000000
100.000000
false

